### PR TITLE
feat: create first version of VPC

### PR DESCRIPTION
**Pull request by Lmam22**
Create first version of VPC:
-2 public subnets
-2 private subnets
-2 db private subnets
-1 transit gateway
-1 nat gateway
-2 route tables

### CODE REVIEW

**Review by pramirezma — APPROVED** [review]

**Review by pramirezma — APPROVED** [review]

File: modules/vpc_manual/natgw.tf [hunk 1]
```diff
@@ -0,0 +1,51 @@
+#6)Create a NAT gateway for private subnets to access the internet
+
+##########--- NAT Gateway ---##########
+
+# Create Elastic IP for the NAT Gateway
+resource "aws_eip" "elasticip_natgw" {
+  #count  = length(var.public_subnet_cidrs)
```
Comment (jose-sal-y-rosas) [inline modules/vpc_manual/natgw.tf:7]: Please remove this line.

File: modules/vpc_manual/natgw.tf [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+}
+
+##########--- RT Private Subnets ---##########
+#Create a new Route Table 1 for Private Subnets
+#Set as main route table, because you can use 
+#NAT Gateway in all the public subnets, but 
+#without main you only can use it with the 
+#subnet where NAT Gateway is
+resource "aws_route_table" "private_subnet_rt" {
+  #count      = length(var.private_subnet_cidrs)
```
Comment (jose-sal-y-rosas) [inline modules/vpc_manual/natgw.tf:32]: Please remove this line

File: modules/vpc_manual/subnets.tf [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+#Public subnets
+resource "aws_subnet" "public_subnets" {
+  count                   = length(var.public_subnet_cidrs) #2
+  vpc_id                  = aws_vpc.vpc_aws.id
+  cidr_block              = element(var.public_subnet_cidrs, count.index)
+  availability_zone       = element(var.azs, count.index)
+  map_public_ip_on_launch = true #This is what makes it a public subnet
+  #Enable automatic public IP assignment on instance launch!
+  tags = {
+    Name = "public-chupito-${count.index + 1}"
```
Comment (jose-sal-y-rosas) [inline modules/vpc_manual/subnets.tf:12]: Name = "public-subnet-${count.index + 1}"
Project = "chupito"

File: modules/vpc_manual/igw.tf [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+resource "aws_route_table" "public_subnet_rt" {
+  vpc_id = aws_vpc.vpc_aws.id
+
+  route {
+    cidr_block = "0.0.0.0/0"
+    gateway_id = aws_internet_gateway.igw_aws.id
+  }
+
+  tags = {
+    Name = "chupito"
```
Comment (jose-sal-y-rosas) [inline modules/vpc_manual/igw.tf:22]: Project = "chupito"

**Review by jose-sal-y-rosas — CHANGES_REQUESTED** [review]
Please review feedback

**Review by jose-sal-y-rosas — APPROVED** [review]

**Pull request merged**